app/static/js/load_weeks.js

```diff
@@ -0,0 +1,29 @@
+const container = document.querySelector('.calender-grid')
```

Comment:
Run these lines only [after the page loads](https://stackoverflow.com/questions/807878/how-to-make-javascript-execute-after-page-load), or the `.calendar-grid`